### DISCUSSION TITLE
A Security Onion ISO with more than 1GB /boot

### DISCUSSION THREAD

**Discussion by TCLSmith** [Security-Onion-Solutions/securityonion#12110] [2.4] [answered]
### Version

2.4.100

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

64

### RAM

256

### Storage for /

500GB

### Storage for /nsm

6TB

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Good Afternoon Everyone. I have been using Security Onion for a while now and like it a lot, but are there any latest Security Onion iso's that have the boot folder larger than 1 GB. I want to have an increased boot folder for our server to increase the amount of kernels we can install on the server. I have tried to increase the boot partition directly but because the system LVM immediately takes the rest of the storage on the system hard drive, any attempt will cause the drive to not boot. So I tried to edit the ISO itself but editing or creating a ISO destroys the partition boot table. Is there any 2.4 iso's with like a 5GB boot partition or something at least higher than 1GB. We are installing this on Bare-metal server so I can not just increase the size of the storage.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by reyesj2** [comment, marked as answer]
You could use a custom kickstart https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/installation_guide/sect-kickstart-howto

Forgot to mention you could also just install Oracle Linux 9 from their ISO and setup your partitions. Then perform a Security Onion network install.

**Reply by TCLSmith** [reply]
The ISO already has a KS file its just if you extract the ISO and try to remake it with the new KS the files the size of ISO is to large and partition table gets messed up. I have been messing with it for weeks now so Im just going to download Ubuntu and do a network install.

**Reply by defensivedepth** [reply]
For context, (if you can share) how many/what type of kernels are you wanting to store?
